includes/forms/class-wp-job-manager-form-submit-job.php

```diff
@@ -173,7 +173,7 @@ public function init_fields() {
 					'label'       => __( 'Job type', 'wp-job-manager' ),
 					'type'        => $job_type,
 					'required'    => true,
-					'placeholder' => '',
+					'placeholder' => __( 'Choose job type&hellip;', 'wp-job-manager' ),
```

Comment:
Just info: more hellip in WP core:
https://github.com/WordPress/WordPress/search?utf8=%E2%9C%93&q=%26hellip&type=